contrib_bots/lib/googlesearch.py

```diff
@@ -0,0 +1,61 @@
+# See readme.md for instructions on running this code.
+from google import search
+
+def get_google_result(original_content):
+    original_content = original_content.split(' ', 1)[1]
+    try:
+        urls = search(original_content, stop=20)
+    except:
```

Comment:
due to the function triage_message, there is already a check to see whether the message starts with @google, this means that the only error that could occur is the users search term (whatever comes after @google) not being on google as users can search for anything on google. Therefore this is the only error that could occur.